packages/core/src/payment/strategies/clearpay/clearpay-payment-strategy.ts

```diff
@@ -122,7 +122,7 @@ export default class ClearpayPaymentStrategy implements PaymentStrategy {
     private async _loadPaymentMethod(gatewayId: string, methodId: string, options?: RequestOptions): Promise<InternalCheckoutSelectors> {
         try {
             return await this._store.dispatch(
-                this._paymentMethodActionCreator.loadPaymentMethod(`${gatewayId}?method=${methodId}`, options)
+                this._paymentMethodActionCreator.loadPaymentMethod(gatewayId, {...options, params: {method: methodId}})
```

Comment:
@josesonora As discussed, could you review clearpay strategy when you have time? Thanks